scripts/plot_scalar_resources.py

```diff
@@ -0,0 +1,137 @@
+r"""
+Inputs
+-------
+resources : str
+    path of input file with data in scalar resources template format as .csv
+target : str
+    path of output file with prepared data as .csv
+
+Outputs
+---------
+matplotlib.figure.Figure
+    plotted scalar resources
+
+Description
+-------------
+The script takes data in the format of the scalar resources template, prepares the
+data and plots the data.
+"""
+import sys
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
+import oemoflex.tools.plots as plots
+from oemoflex.tools.plots import colors_odict
+
+if __name__ == "__main__":
+    resources = sys.argv[1]
+
+    target = sys.argv[2]
+
+    target_dir = os.path.dirname(target)
+
+    # create the directory plotted where all plots are saved
+    if not os.path.exists(target_dir):
+        os.makedirs(target_dir)
+
+    def load_scalars(path):
+        df = pd.read_csv(path, sep=",", index_col=0)
+        return df
+
+    # Load scalar data
+    scalars = load_scalars(resources)
+
+    # GENERAL
+    unit_dict = {"capacity_net_el": "W"}
+    carrier_dict = {
+        "biomass": "Biomass",
+        "ch4": "CH4",
+        "hard coal": "Hard coal",
+        "oil": "Oil",
+        "lignite": "Lignite",
+        "other": "Other",
+    }
+
+    german_labels = {
+        "Biomass": "Biomasse",
+        "CH4": "Erdgas",
+        "Hard coal": "Steinkohle",
+        "Oil": "Öl",
+        "Lignite": "Braunkohle",
+        "Other": "Andere",
+    }
+
+    # User input
+    var_name = "capacity_net_el"
+    conv_number = 1000 * 1000
```

Comment:
This is a bit obscure. An inline comment would help (converting Megawatt to Watt, I guess?). Also, should just write 1e6.